extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalFooter_index.php

```diff
@@ -11,6 +11,13 @@
 					) ?></span>
 			</h2>
 		</div>
+	<?php elseif ( isset ( $model['header'] ) ) : ?>
+		<h2 class="wds-global-footer__header">
+			<?= DesignSystemHelper::getSvg(
+				$model['header']['image'],
+				'wds-global-footer__header-logo'
+			) ?>
+		</h2>
 	<?php endif; ?>
 	<div class="wds-global-footer__main">
 		<?php if ( isset( $model['fandom']['header'] ) ) : ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "fandom" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)